docs/rules/object-curly-spacing.md

```diff
@@ -22,28 +22,28 @@ export { foo };
 
 ## Rule Details
 
-This rule aims to maintain consistency around the spacing inside of object literals. It also
-applies to EcmaScript 6 destructured assignment and import/export specifiers.
-
-It either requires or disallows spaces between those braces and the values inside of them.
-Braces that are separated from the adjacent value by a new line are exempt from this rule.
+This rule enforce consistent spacing inside braces of object literals, destructuring assignments, and import/export specifiers.
 
 ## Options
 
-There are two main options for the rule:
+This rule has a string option:
```

Comment:
Why not condense that to "This rule has a string option and an object option" so that it's consistent and makes more sense to say "This rule has two string options" when the options are of the same type? Seems @pedrottimark has been busy, but I'd like to know what he thinks as well since the guidelines would need to be updated.